graphenv/vertex.py

```diff
@@ -78,7 +78,7 @@ def _make_observation(self) -> any:
     def children(self) -> List[V]:
         """
         Gets the child vertices of this vertex.
-        Acts as a wrapper that memoizes calls to _get_children() and
+        Acts as a wrapper that memorizes calls to _get_children() and
```

Comment:
I suspect either works: https://en.wikipedia.org/wiki/Memoization
(happy to keep the change though)

Reply:
Oh, you are right. This is a more technical word. I prefer it! It's up to you